gpii/node_modules/flowManager/src/userKeyaction.js

```diff
@@ -0,0 +1,221 @@
+/*
+ * GPII User Login Component
```

Comment:
I think a name like userLoginChange or userLogonStateChange is more descriptive - there's nothing about the "key" which is acting. Unless we are going to make a wholesale renaming throughout the architecture of "login" to "keyin" etc.